doc/versions.json

```diff
@@ -19,6 +20,12 @@
       "nrfxlib": "1.4.99",
       "zephyr": "2.4.0",
       "mcuboot": "1.6.99"
+    },
+      "1.4.2": {
+      "ncs": "1.4.2",
+      "nrfxlib": "1.4.0",
```

Comment:
nrfxlib must be (at least) 1.4.1 - that was missed in the update for 1.4.1, but is fixed on master (see https://github.com/nrfconnect/sdk-nrf/pull/3570 ).
This must also be fixed in the conf.py file for nrfxlib.

Reply:
agree

Reply:
Resolved but not fixed?

This is needed here and in nrf/doc/nrfxlib/conf.py